### PR TITLE
Change to V3.0.3

### PR DESCRIPTION
**Pull request by BirgitBoss**

### CODE REVIEW

File: documentation/IDTA-01002-3/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 This specification is part of the https://industrialdigitaltwin.org/en/content-hub/aasspecifications[Asset Administration Shell Specification series].
 ====
 
-NOTE: We do not provide a web version of the specification for version 3.0.2.
+NOTE: We do not provide a web version of the specification for version 3.0.3
 Please download and use the PDF version of the document instead.
 
-link:https://industrialdigitaltwin.org/en/wp-content/uploads/sites/2/2024/06/IDTA-01002-3-0-2_SpecificationAssetAdministrationShell_Part2_API.pdf[Download the specification in PDF format]
\ No newline at end of file
+link:https://industrialdigitaltwin.org/en/wp-content/uploads/sites/2/2024/06/IDTA-01002-3-0-3_SpecificationAssetAdministrationShell_Part2_API.pdf[Download the specification in PDF format]
```
Comment (BirgitBoss) [inline documentation/IDTA-01002-3/modules/ROOT/pages/index.adoc:12]:
```suggestion
link:https://industrialdigitaltwin.org/en/wp-content/uploads/sites/2/2024/10/IDTA-01002-3-0-3_SpecificationAssetAdministrationShell_Part2_API.pdf[Download the specification in PDF format]
```

**Comment by BirgitBoss** [comment]
Do we need separate branch for V3.0.3?

TBD: Label needs to be changed.

**Review by sebbader-sap — APPROVED** [review]

**Comment by zrgt** [comment]
> Do we need separate branch for V3.0.3?
> 
> TBD: Label needs to be changed.

We need either rename the branch or create a new one. Otherwise it is misleading.

The label should be changed in `documentation/IDTA-01002-3/antora.yml`

**Comment by sebbader-sap** [comment]
What's the purpose of the branch `IDTA-01002-3-0-2_antora_ref`? Honestly speaking, I thought this PR was targeting [IDTA-01002-3-1_preparation](https://github.com/admin-shell-io/aas-specs-api/tree/IDTA-01002-3-1_preparation).

As V3.1.0 will be the first one providing also the specification text, I'd say all non-OpenAPI V3.0.2 or V3.0.3 content lives only in the respective Word file, not here...

**Pull request merged**